lib/node_modules/@stdlib/stats/strided/wasm/dnanvariancewd/README.md

```diff
@@ -0,0 +1,308 @@
+
+var x = new Float64Array( [ 1.0, -2.0, NaN, 2.0 ] );
+
+var v = dnanvariancewd.main( x.length, 1, x, 1 );
+// returns ~4.3333
+```
+
+The function has the following parameters:
+
+-   **N**: number of indexed elements.
+-   **correction**: degrees of freedom adjustment. Setting this parameter to a value other than `0` has the effect of adjusting the divisor during the calculation of the [variance][variance] according to `n-c` where `c` corresponds to the provided degrees of freedom adjustment and `n` corresponds to the number of non-`NaN` indexed elements. When computing the [variance][variance] of a population, setting this parameter to `0` is the standard choice (i.e., the provided array contains data constituting an entire population). When computing the unbiased sample [variance][variance], setting this parameter to `1` is the standard choice (i.e., the provided array contains data sampled from a larger population; this is commonly referred to as Bessel's correction).
+-   **x**: input [`Float64Array`][@stdlib/array/float64].
+-   **strideX**: stride length for `x`.
+
+The `N` and stride parameters determine which elements in the strided array are accessed at runtime. For example, to access every other element in `x`,
+
+<!-- eslint-disable max-len -->
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var x = new Float64Array( [ 1.0, 2.0, 2.0, -7.0, -2.0, 3.0, 4.0, 2.0, NaN, NaN ] );
+
+var v = dnanvariancewd.main( 5, 1, x, 2 );
+// returns 6.25
+```
+
+Note that indexing is relative to the first index. To introduce an offset, use [`typed array`][mdn-typed-array] views.
+
+<!-- eslint-disable stdlib/capitalized-comments, max-len -->
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var x0 = new Float64Array( [ 2.0, 1.0, 2.0, -2.0, -2.0, 2.0, 3.0, 4.0, NaN, NaN ] );
+var x1 = new Float64Array( x0.buffer, x0.BYTES_PER_ELEMENT*1 ); // start at 2nd element
+
+var v = dnanvariancewd.main( 5, 1, x1, 2 );
+// returns 6.25
+```
+
+#### dnanvariancewd.ndarray( N, correction, x, strideX, offsetX )
+
+Computes the [variance][variance] of a double-precision floating-point strided array ignoring `NaN` values and using Welford's algorithm and alternative indexing semantics.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var x = new Float64Array( [ 1.0, -2.0, NaN, 2.0 ] );
+
+var v = dnanvariancewd.ndarray( x.length, 1, x, 1, 0 );
+// returns ~4.3333
+```
+
+The function has the following additional parameters:
+
+-   **offsetX**: starting index for `x`.
+
+While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying buffer, the offset parameter supports indexing semantics based on a starting index. For example, to access every other element starting from the second element:
+
+<!-- eslint-disable max-len -->
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+// eslint-disable-next-line max-len
```

Comment:
```suggestion
```

Reply:
@0PrashantYadav0 In README examples, we shouldn't be showing ESLint disable comments. These should be added as HTML comments before the code block.